olive/passes/onnx/moe_experts_distributor.py

```diff
@@ -363,6 +364,12 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
                 required=True,
                 description=("Number of GPU nodes to distribute the model for. Must be greater than 1."),
             ),
+            "jobs": PassConfigParam(
+                type_=int,
```

Comment:
Just name it parallel jobs.
Also in the description clarify that default value is no of cpus.
Looks good otherwise.

Reply:
Renamed it to "parallel_jobs".